src/core/README.md

```diff
@@ -138,15 +138,15 @@ See [LICENSE](LICENSE) for details.
 
 <!-- doc-viewer-config
 {
-	"api": "docs/api.json",
+	"api": "../../docs/core/api.json",
```

Comment:
Can we remove the relative paths to these files and to the pages? Relative to the top-level of the repository would be consistent with how the other repos do it.